opencti-platform/opencti-graphql/src/database/redis.ts

```
@@ -123,6 +123,14 @@ const sentinelOptions = async (clusterNodes: Partial<SentinelAddress>[]): Promis
     enableTLSForSentinelMode: conf.get('redis:sentinel_tls') ?? false,
     failoverDetector: conf.get('redis:sentinel_failover_detector') ?? false,
     updateSentinels: conf.get('redis:sentinel_update_sentinels') ?? true,
+    sentinelReconnectStrategy: (retryAttempts: number) => {
+      const sentinelMaxReconnect = conf.get('redis:sentinel_max_reconnects');
+      if (!!sentinelMaxReconnect && retryAttempts >= sentinelMaxReconnect) {
+        logApp.error('Shutting down: redis sentinel was not able to reconnect');
+        process.exit(1);
+      }
+      return 60000;
+    },
```

is it still relevant ?
This concept of "fail fast" is nowhere else in the app yet, and it might be breaking change for some people, depending on how they manage their platform infra.
cc @richard-julien

---

removing for now since it is not a necessary change